### ISSUE TITLE
Cleanup Pull-Requests

### ISSUE THREAD

**Issue by lazaridis-com** [geddy/geddy#732]
Prior to getting new people on board, the old PRs (older than 3 months) should be processed. Usually this means that they should be closed (by the last active maintainers). I reviewed some of them, and they contain far too much unstructured information.

**Comment by ckhatton** [comment]
Yeah, I think it's more that everyone is too busy. But yes, I agree they need processing.
:squid:

**Comment by mde** [comment]
Good call. I am happy to help triage, if someone will work with me to keep the ball rolling. :)

**Comment by lazaridis-com** [comment]
@mde, @ckhatton - this is more kind of a RESET. The PRs are so old, the should be simply closed (one can dig information out, if needed in future).
